src/test/java/pl/marcin/project/repository/CustomerRepositoryListBasedTest.java

```diff
@@ -38,7 +40,166 @@ public void shouldReturnCustomerWithId2() {
 
         //then
         Assertions.assertEquals(id2, customer.getId());
+    }
+
+    @Test
+    public void shouldThrowExceptionIfNotFindCustomerById() {
+        //given
+        int id = 1;
+        int id2 = 2;
+        //when:
+        Customer customer2 = Customer.builder().id(id2).build();
+        customerRepository.saveCustomer(customer2);
+
+        //then:
+        assertThrows(RuntimeException.class, () -> {
+            customerRepository.findCustomer(id);
+        });
+    }
+
+    @Test
+    public void shouldThrowExceptionMessageNoSuchCustomerById() {
```

Comment:
this test with this one `shouldThrowExceptionIfNotFindCustomerById` can be merged

Reply:
ok